NLP/word2vec/README.md

```diff
@@ -0,0 +1,81 @@
+# Tensorflow Implementation of Word2Vec (Dataset from kaggle file)
+
+## Usage
+### To train
```

Comment:
you can also add help log for indicating available arguments

Reply:
is it necessary

Reply:
done